### PR TITLE
Stacked and multi stacked bar accessibility

### PR DESCRIPTION
**Pull request by shivasai09**
#### Pull request checklist

-[x] https://uifabric.visualstudio.com/iss/_workitems/edit/1164 
- [x] Include a change request file using `$ yarn change`

#### Description of changes
when we are focusing the data-segment of the chart we are reading information present in callout .

#### Focus areas to test
stacked bar chart and multistacked bar chart

#### video of stacked bar chart
https://drive.google.com/file/d/1c4tpc-RGMYY25o4M5ZrB_Sv8Lf4elWiE/view?usp=sharing

#### video of multistacked bar chart
https://drive.google.com/file/d/1HTElK8BFEQETBBEdzcfyxk03gh9S9MjZ/view?usp=sharing

NOTE:- please download the videos from the above link if they aren't processed 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/12217)

### CODE REVIEW

File: packages/charting/src/components/StackedBarChart/StackedBarChart.base.tsx [hunk 1]
```diff
@@ -140,6 +140,7 @@ export class StackedBarChartBase extends React.Component<IStackedBarChartProps,
                   target={this.state.refSelected}
                   setInitialFocus={true}
                   directionalHint={DirectionalHint.topRightEdge}
+                  id={'callout'}
```
Comment (dzearing) [inline packages/charting/src/components/StackedBarChart/StackedBarChart.base.tsx]: Don't hardcode an ID... what if you have 2 on the same page?

Just use `getId` in utilities to generate a unique ID and cache it in a member (this._calloutId) and use that.

Comment (shivasai09) [reply packages/charting/src/components/StackedBarChart/StackedBarChart.base.tsx]: done

**Comment by shivasai09** [comment]
@aneeshack4  there are no examples files to get updated. it is accessibility thing and internally all the examples are using this file.. you could see the after fix video mentioned in the description

File: change/@uifabric-charting-2020-03-06-11-00-41-user-v-sivsar-StackedAndMultiStackedbarAccesiblity.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
+  "comment": "fixed accessiblity for the  statcked and multistacked bar chart",
```
Comment (aneeshack4) [inline change/@uifabric-charting-2020-03-06-11-00-41-user-v-sivsar-StackedAndMultiStackedbarAccesiblity.json]: Please polish the commit message as these go out to our customers in release notes. Suggestion "StackedBarChart: added id for each callout and setting aria-labelledby accordingly"

Comment (shivasai09) [reply change/@uifabric-charting-2020-03-06-11-00-41-user-v-sivsar-StackedAndMultiStackedbarAccesiblity.json]: done

**Pull request merged**